test/integration/wallet.test.js

```diff
@@ -391,7 +391,7 @@ describe('@integration: Wallet', () => {
       step('Then I should be redirected to transactoinDetails step', () => helper.checkRedirectionToDetails('123456'));
     });
 
-    describe('Scenario: should allow to filter transactions', () => {
+    describe.skip('Scenario: should allow to filter transactions', () => {
```

Comment:
Please add a comment saying "This should be unskipped in #<ISSUE_NUMBER>"

Reply:
Comment fixed with issue number added